tests/integrations/celery/test_celery.py

```diff
@@ -509,7 +509,8 @@ def test_baggage_propagation(init_celery):
     def dummy_task(self, x, y):
         return _get_headers(self)
 
-    with start_transaction() as transaction:
+    # force trace_id for predictable sample_rand
+    with start_transaction(trace_id="00000000000000000000000000000000"):
```

Comment:
not relevant for this PR, but with potel we will no longer support manual `trace_id` so maybe you can already use a mock here instead